books/spiders/books_list.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from enum import Enum
+
+
+@dataclass
+class Book:
+    title: str
+    price: float
+    rating: int
+    amount_in_stock: str
+    category: str
+    description: str
+    upc: str
+
+
+class RatingEnum(Enum):
+    Zero = 0
+    One = 1
+    Two = 2
+    Three = 3
+    Four = 4
+    Five = 5
+
+
+def get_rating_value_from_class_names(class_names: str) -> int | None:
+    class_names_list = class_names.split()
+    rating_value_class = class_names_list[-1]
+    return RatingEnum[rating_value_class].value
```

Comment:
The `get_rating_value_from_class_names` function assumes that the last class name corresponds to a valid `RatingEnum` key. Ensure that the input `class_names` always contains a valid rating class name to avoid `KeyError`. Consider adding error handling for cases where the class name might not match any `RatingEnum` key.